package.json

```diff
@@ -1,214 +1,214 @@
-		"mockdate": "^3.0.5",
-		"object-fit-videos": "^1.0.3",
-		"ophan-tracker-js": "1.4.0",
-		"ora": "^1.3.0",
-		"pify": "^5.0.0",
-		"postcss": "^8.3.10",
-		"postcss-pxtorem": "^6.0.0",
-		"preact": "^10.5.13",
-		"prettier": "^2.2.1",
-		"prettysize": "^0.1.0",
-		"qwery": "3.4.2",
-		"rangefix": "^0.2.5",
-		"raven-js": "^3.19.1",
-		"raw-loader": "^0.5.1",
-		"request": "^2.81.0",
-		"requireindex": "^1.1.0",
-		"rimraf": "^2.6.1",
-		"rxjs": "^6.6.7",
-		"sass": "1.77.8",
-		"sass-graph": "4.0.1",
-		"sass-loader": "14.2.1",
-		"sass-mq": "5.0.1",
-		"semver": "^5.4.1",
-		"split": "^1.0.0",
-		"stream-to-observable": "^0.2.0",
-		"stylelint": "^16.5.0",
-		"stylelint-config-standard-scss": "^13.1.0",
-		"svgo": "^2.3.0",
-		"tcp-ping": "^0.1.1",
-		"ts-jest": "29.1.2",
-		"ts-loader": "^8.0.12",
-		"tslib": "2.6.2",
-		"typescript": "5.3.3",
-		"uglify-js": "^3.19.2",
-		"video.js": "~5.3",
-		"videojs-contrib-ads": "3.1.3",
-		"videojs-embed": "guardian/videojs-embed#v0.3.3",
-		"videojs-ima": "googleads/videojs-ima#0.3.0",
-		"videojs-persistvolume": "guardian/videojs-persistvolume#0.1.4",
-		"videojs-playlist": "guardian/videojs-playlist#0.1.4",
-		"web-vitals": "3.5.1",
-		"webpack": "^5.79.0",
-		"webpack-bundle-analyzer": "^4.10.2",
-		"webpack-cli": "^5.0.1",
-		"webpack-dev-server": "^5.0.4",
-		"webpack-merge": "^4.2.2",
-		"wolfy87-eventemitter": "~5.2.4",
-		"yargs": "^17.7.1"
-	},
-	"optionalDependencies": {
-		"fsevents": "^2.3.2",
-		"react": "18.3.1",
-		"react-dom": "18.2.0"
-	},
-	"jest": {
-		"preset": "ts-jest/presets/js-with-babel",
-		"moduleFileExtensions": [
-			"ts",
-			"tsx",
-			"js"
-		],
-		"transform": {
-			"^.+\\.(ts|tsx)$": [
-				"ts-jest",
-				{
-					"tsconfig": "tsconfig.test.json"
-				}
-			]
-		},
-		"testMatch": [
-			"**/*.(test|spec).+(ts|tsx|js)"
-		],
-		"roots": [
-			"<rootDir>/static/src/javascripts",
-			"<rootDir>/tools"
-		],
-		"moduleDirectories": [
-			"<rootDir>/static/src/javascripts",
-			"<rootDir>/static/src/javascripts/projects",
-			"<rootDir>/static/vendor/javascripts",
-			"node_modules"
-		],
-		"moduleNameMapper": {
-			"videojs": "video.js",
-			"^svgs/(.*)$": "<rootDir>/__mocks__/svgMock.js",
-			"^(.*)\\.svg$": "<rootDir>/__mocks__/svgMock.js",
-			"^(.*)\\.html$": "<rootDir>/__mocks__/templateMock.js",
-			"ophan/ng": "ophan-tracker-js",
-			"ophan/embed": "ophan-tracker-js/build/ophan.embed"
-		},
-		"setupFilesAfterEnv": [
-			"<rootDir>/dev/jest.setupTestFrameworkScriptFile.js"
-		],
-		"testEnvironment": "jest-environment-jsdom-global",
-		"testEnvironmentOptions": {
-			"url": "http://testurl.theguardian.com"
-		},
-		"transformIgnorePatterns": [
-			"/node_modules/(?!(@guardian|lodash-es|preact)/)"
-		]
-	},
-	"scripts": {
-		"eslint-fix": "eslint --color --fix",
-		"test": "jest"
-	},
-	"prettier": "@guardian/prettier",
-	"packageManager": "yarn@4.1.1",
-	"resolutions": {
-		"micromatch": "4.0.8",
-		"unset-value": "^2.0.1"
-	}
+  "name": "@guardian/frontend",
```

Comment:
An accidental reformat?

Reply:
Must have snuck in when I was rebasing 🤦‍♀️ will fix it!